Makefile

```diff
@@ -219,17 +219,31 @@ check-kuttl:
 
 .PHONY: generate-kuttl
 generate-kuttl: export KUTTL_PG_VERSION ?= 14
-generate-kuttl: export KUTTL_PSQL_IMAGE ?= registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+generate-kuttl: export KUTTL_PSQL_IMAGE ?= registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.2-0
 generate-kuttl:
 	[ ! -d testing/kuttl/e2e-generated ] || rm -r testing/kuttl/e2e-generated
 	bash -ceu ' \
-	render() { envsubst '"'"'$$KUTTL_PG_VERSION $$KUTTL_PSQL_IMAGE'"'"'; }; \
+	render() { envsubst '"'"'$$KUTTL_PG_VERSION $$KUTTL_POSTGIS_VERSION $$KUTTL_PSQL_IMAGE'"'"'; }; \
 	while [ $$# -gt 0 ]; do \
 		source="$${1}" target="$${1/e2e/e2e-generated}"; \
 		mkdir -p "$${target%/*}"; render < "$${source}" > "$${target}"; \
 		shift; \
 	done' - $(wildcard testing/kuttl/e2e/*/*.yaml)
 
+.PHONY: generate-kuttl-other
+generate-kuttl-other: export KUTTL_PG_VERSION ?= 14
```

Comment:
I don't like to see this duplicated but it's fine. I do wonder if we could have something like this :recycle:
```
kuttl-envs: export BLAH
kuttl-envs: export BLAB

generate-kuttl:  kuttl-envs
<stuff>

generate-kuttl-other: kuttl-envs
generate-kuttl-other: export BLAH-other
<other stuff>
```

Reply:
I tried something similar, but unfortunately couldn't get the syntax correct. Maybe there's an another way to do this? For now, I'll leave this as is.

Reply:
🔧 I recommend combining these so we have only the `generate-kuttl` target.

```diff
diff --git Makefile Makefile
index 41cb2804d..0de84393b 100644
--- Makefile
+++ Makefile
@@ -219,22 +219,10 @@ check-kuttl:
 
 .PHONY: generate-kuttl
 generate-kuttl: export KUTTL_PG_VERSION ?= 14
+generate-kuttl: export KUTTL_POSTGIS_VERSION ?= 3.1
 generate-kuttl: export KUTTL_PSQL_IMAGE ?= registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.2-0
 generate-kuttl:
 	[ ! -d testing/kuttl/e2e-generated ] || rm -r testing/kuttl/e2e-generated
-	bash -ceu ' \
-	render() { envsubst '"'"'$$KUTTL_PG_VERSION $$KUTTL_PSQL_IMAGE'"'"'; }; \
-	while [ $$# -gt 0 ]; do \
-		source="$${1}" target="$${1/e2e/e2e-generated}"; \
-		mkdir -p "$${target%/*}"; render < "$${source}" > "$${target}"; \
-		shift; \
-	done' - $(wildcard testing/kuttl/e2e/*/*.yaml)
-
-.PHONY: generate-kuttl-other
-generate-kuttl-other: export KUTTL_PG_VERSION ?= 14
-generate-kuttl-other: export KUTTL_POSTGIS_VERSION ?= 3.1
-generate-kuttl-other: export KUTTL_PSQL_IMAGE ?= registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.2-0
-generate-kuttl-other:
 	[ ! -d testing/kuttl/e2e-generated-other ] || rm -r testing/kuttl/e2e-generated-other
 	bash -ceu ' \
 	render() { envsubst '"'"'$$KUTTL_PG_VERSION $$KUTTL_POSTGIS_VERSION $$KUTTL_PSQL_IMAGE'"'"'; }; \
@@ -242,7 +230,7 @@ generate-kuttl-other:
 		source="$${1}" target="$${1/e2e/e2e-generated}"; \
 		mkdir -p "$${target%/*}"; render < "$${source}" > "$${target}"; \
 		shift; \
-	done' - $(wildcard testing/kuttl/e2e-other/*/*.yaml)
+	done' - $(wildcard testing/kuttl/e2e/*/*.yaml) $(wildcard testing/kuttl/e2e-other/*/*.yaml)
 
 .PHONY: check-generate
 check-generate: generate-crd generate-deepcopy generate-rbac
```

Reply:
:+1: That works for me, the generation is pretty fast regardless.